Demo/Demo/SwiftUIComponents/CardPaymentViews/CardOrderActionButton.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+struct CardOrderActionButton: View {
+
+    let intent: String
+    let orderID: String
+    let selectedMerchantIntegration: MerchantIntegration
+
+    @ObservedObject var cardPaymentViewModel: CardPaymentViewModel
+
+    var body: some View {
+        ZStack {
+            Button("\(intent)") {
+                if intent == "CAPTURE" {
```

Comment:
nit: Could we move this logic into a `private func completeOrder(intent: String) async` method? The goal would be to cleanly separate SwiftUI rendering logic from the business logic.

Reply:
For reference, we do something similar on Android in [CardFragment.kt](https://github.com/paypal/paypal-android/blob/4342b55e1f65dd3821ac89c81c0f8198d1df71d8/Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt#L156).

Reply:
Oh that reminds me Steven, we forgot to add some device collection logic for iOS. I think we were going to do a follow up PR.

Reply:
Moved to private function in this commit: 314b65661c24dd185563e3a5a7ac9cd27081af4f